backend-demo/README.md

```diff
@@ -0,0 +1,16 @@
+# Portal backend demo
+
+ASHPD enables writing [portal backends](https://docs.flatpak.org/en/latest/portal-api-reference.html#idm45771421251808) that implement a subset of backend interfaces.
+
+This demo provides a backend that registers the `org.freedesktop.impl.portal.desktop.ashpd-backend-demo` name and implements the [Account](https://docs.flatpak.org/en/latest/portal-api-reference.html#gdbus-org.freedesktop.impl.portal.Account), [Screenshot](https://docs.flatpak.org/en/latest/portal-api-reference.html#gdbus-org.freedesktop.impl.portal.Screenshot), [Secret](https://docs.flatpak.org/en/latest/portal-api-reference.html#gdbus-org.freedesktop.impl.portal.Secret), [Settings](https://docs.flatpak.org/en/latest/portal-api-reference.html#gdbus-org.freedesktop.impl.portal.Settings) and [Wallpaper](https://docs.flatpak.org/en/latest/portal-api-reference.html#gdbus-org.freedesktop.impl.portal.Wallpaper) interfaces.
+
+Note that for the portal frontend to find and use a backend, the former needs to be made aware of the existence of the latter by way of [a portal file](https://flatpak.github.io/xdg-desktop-portal/docs/writing-a-new-backend.html#portal-portal-file) and [a configuration file](https://flatpak.github.io/xdg-desktop-portal/docs/configuration-file.html).
+These files are typically installed under `{DATADIR}/xdg-desktop-portal/`, but for testing purposes the `$XDG_DESKTOP_PORTAL_DIR` environment variable can be set to instruct the frontend where to look for them, e.g.:
+
+```shell
+XDG_DESKTOP_PORTAL_DIR=$PWD/backend-demo/data /usr/libexec/xdg-desktop-portal -v -r
+```
+
+Note that this will temporarily override your system's defaults, so other existing backends (gnome, kde, …) won't be used as fallbacks, and as a consequence interfaces not implemented by this demo backend won't be available for applications to use.
```

Comment:
```suggestion
Note that this will temporarily override your system's defaults, so other existing backends (GNOME, KDE, …) won't be used as fallbacks, and as a consequence interfaces not implemented by this demo backend won't be available for applications to use.
```